### PR TITLE
feat(fix) : Pagination with only one page inside

### PR DESCRIPTION
**Pull request by ravi-kolla**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -396,13 +396,17 @@ export class Carousel {
...
 		const {paginationTemplate, paginationClassName, paginationLabel, paginationTitle} = _options;
-		const pagination = __render(paginationTemplate, {
-			pages,
-			controls: id,
-			className: paginationClassName,
-			label: paginationLabel,
-			title: paginationTitle,
-		});
+		if (pages.length > 1) {
+			const pagination = __render(hasPagination, {
```
Comment (schorfES) [inline src/index.js:400]: The render function expects a template. You're passing the setting if the pagination feature should be enabled.

```suggestion
			const pagination = __render(paginationTempate, {
```

File: src/index.js [hunk 2]
```diff
@@ -396,13 +396,17 @@ export class Carousel {
...
+				pages,
+				controls: id,
+				className: paginationClassName,
+				label: paginationLabel,
+				title: paginationTitle,
+			});
+		} else {
```
Comment (schorfES) [inline src/index.js:407]: I would suggest to handle it codewise a bit differently. There is no need for a if/else statement.

```
if (pages.length < 2) {
  return;
}
```

**Review by schorfES — CHANGES_REQUESTED** [review]
Hi @ravi-kolla and thank you for your contribution. ❤️ 

The PR is currently failing because of some detected security vulnerabilities during the CI validation and tests. I am going to fix these errors.

I am currently working on a new release. These new features/updates are located in a [feature/next](https://github.com/schorfES/caroucssel/tree/feature/next). Can you please use that branch as a basis for you changes right now?

Can you also add some tests for this fix to test if the rendering of the pagination runs properly? I think about these test cases:

* Renders no pagination if a pagination has only one page
* Re-renders the pagination when a the amount of pages changes (by viewport/item size changes)
> * from state rendered with pagination to state without
> * from state rendered without pagination to state with

**Comment by schorfES** [comment]
Hi @ravi-kolla, I've added the described changes and merged you commit's into the `feature/next` branch. Thank you for your contribution.

**Comment by ravi-kolla** [comment]
Thank you @schorfES 👍

**Comment by schorfES** [comment]
There is a pre-release available at npm using caroucssel@next

**Pull request closed without merging**